common/libzkp/impl/src/batch.rs

```diff
@@ -148,11 +149,33 @@ pub unsafe extern "C" fn gen_batch_proof(
 
 /// # Safety
 #[no_mangle]
-pub unsafe extern "C" fn verify_batch_proof(proof: *const c_char) -> c_char {
+pub unsafe extern "C" fn verify_batch_proof(
+    proof: *const c_char,
+    fork_name: *const c_char,
+) -> c_char {
     let proof = c_char_to_vec(proof);
     let proof = serde_json::from_slice::<BatchProof>(proof.as_slice()).unwrap();
-
-    let verified = panic_catch(|| VERIFIER.get().unwrap().verify_agg_evm_proof(proof));
+    let fork_name_str = c_char_to_str(fork_name);
+    let fork_id = match fork_name_str {
+        "" => 1,
+        "shanghai" => 1,
+        "bernoulli" => 2,
```

Comment:
I'd suggest using `0` and `1` for consistency (as we do in `codecv0` and `codecv1`, and also in the batch header `Version` field).

Reply:
i think we use names like "upgrade#1" "upgrade#2" "upgrade#3".  I feel being consistent with that is good?

Reply:
Well "upgrade #1" has no significance to provers/verifiers. We're already using 0/1 everywhere (except for the names you mentioned). But we can also keep this PR as it is.

Reply:
“using 0/1 everywhere”: in l2geth？then i think it may be better to be consistent

Reply:
Yes https://github.com/scroll-tech/go-ethereum/blob/develop/rollup/rollup_sync_service/rollup_sync_service.go#L469